xbmc/addons/PluginSource.cpp

```diff
@@ -80,29 +86,11 @@ CPluginSource::Content CPluginSource::Translate(const std::string &content)
     return CPluginSource::UNKNOWN;
 }
 
-TYPE CPluginSource::FullType() const
-{
-  if (Provides(VIDEO))
-    return ADDON_VIDEO;
-  if (Provides(AUDIO))
-    return ADDON_AUDIO;
-  if (Provides(IMAGE))
-    return ADDON_IMAGE;
-  if (Provides(GAME))
-    return ADDON_GAME;
-  if (Provides(EXECUTABLE))
-    return ADDON_EXECUTABLE;
-
-  return CAddon::FullType();
-}
-
-bool CPluginSource::IsType(TYPE type) const
+bool CPluginSource::HasType(TYPE type) const
 {
-  return ((type == ADDON_VIDEO && Provides(VIDEO))
-       || (type == ADDON_AUDIO && Provides(AUDIO))
-       || (type == ADDON_IMAGE && Provides(IMAGE))
-       || (type == ADDON_GAME && Provides(GAME))
-       || (type == ADDON_EXECUTABLE && Provides(EXECUTABLE)));
+  return ((type == ADDON_VIDEO && Provides(VIDEO)) || (type == ADDON_AUDIO && Provides(AUDIO)) ||
+          (type == ADDON_IMAGE && Provides(IMAGE)) || (type == ADDON_GAME && Provides(GAME)) ||
+          (type == ADDON_EXECUTABLE && Provides(EXECUTABLE)) || (type == CAddon::Type()));
```

Comment:
Should we keep these on separate lines for readability and git history niceness?

Reply:
Aha github is working again :smile: 
No problem hold it also for better like before was only about the guidelines. 
I change it.